### ISSUE TITLE
Export to Substance default folder

### ISSUE THREAD

**Issue by dayelov** [pleprince/RfSP#7]
Hi there,
small post dedicated to a default Substance export localization. By default textures are exported to 
Substance Painter export folder on local drive.

Guys in my studio starts to export textures, and basically work with assets and save them. After I try to open asset made by other artist on my machine and try export textures, an error comes out. And this is because Substance saves user export path with substance project itself, so instead of C:\Users\mymachine\... you will have C:\Users\JohnDoe\....

So the one thing in mind is to make some temp folder on a network drive, so everybody can have ecces to it and export textures before conversion to that path. In this scenario one path saves with all substance projects which allows to freely share assets in team. That can work.

But if you start to work on a new asset, the path is again set to local export folder by default. And in this case you should manually make a first export to chosen location on a network drive and save your project.

So basically to avoid all those steps we can add our path to a renderman.js script or to a settings.ini file and read it in script as exportPath = osPath(alg.settings.value("tempDir")). It's good when you work alone but required a little more tweaking when you work with team)

Also I've add UDIM support, you can see it in action
https://youtu.be/vRY6w9zhimE

**Comment by pleprince** [comment]
Good point. Do you want make a pull request for that and the UDIm support ?

**Comment by pleprince** [comment]
fixed
